src/main/java/webserver/Response.java

```diff
@@ -0,0 +1,113 @@
+package webserver;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class Response {
+    private static final String CRLF = "\r\n";
+    private static Map<Integer, String> responseStatus = buildResponseCodes();
+    private PrintWriter out;
+    private String path;
+    private int statusCode;
+    private String body = "";
+    private String requestMethod;
+
+
+    public Response(PrintWriter out, String path, String requestMethod) {
+        this.out = out;
+        this.path = path;
+        this.requestMethod = requestMethod;
+    }
+
+    private static Map<Integer, String> buildResponseCodes() {
+        Map<Integer, String> responses = new HashMap<Integer, String>();
+        responses.put(200, "OK");
+        responses.put(404, "Not Found"); 
+        return responses;
+    }
+
+    public void setupDataToBeSent() throws IOException {
+        Path parsedPath = createFilePath(createPathString(path));
+        setStatusCode(parsedPath);
+        if (requestMethod.equals("GET")){
+            getHTML(parsedPath);
+        }
+    }
+
+    public void send() throws IOException {
+        out.print(createInitialResponseLine(statusCode));
+        out.print(CRLF);
+        out.print("Content-Length: " + body.length());
+        out.print(CRLF);
+        out.print("Content-Type: text/html; charset=utf˝-8");
+        out.print(CRLF);
+        out.print(CRLF);
+        out.println(body);
+    }
+
+    private String createPathString(String path) {
+        if (path.equals("/")) {
+            path = "/index.html";
+        }
+
+        if(!path.contains("html")) {
```

Comment:
the spacing for the if on this line and 59 seem different. I think 59 is the standard one??? Not 100% sure

Reply:
Also, I'm a bit confused about this block of code. It seems like a hack to get something working and I think there may be a better way to do this.

Reply:
Yea I don't like this either. I think I need to look into File systems stuff more. How would I find a file when the request path is something like "/todo" without the .html?

Reply:
I'd write something like a router where for certain paths, it triggers something like a controller that will render a page (in this case, from an html file). For me, it'd be something like https://expressjs.com/en/guide/routing.html